server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/comment/review/ReviewCommentConverter.java

```diff
@@ -0,0 +1,21 @@
+package de.tum.in.www1.hephaestus.codereview.comment.review;
+
+import org.kohsuke.github.GHPullRequestReviewComment;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Component;
+
+import de.tum.in.www1.hephaestus.codereview.base.BaseGitServiceEntityConverter;
+
+@Component
+public class ReviewCommentConverter extends BaseGitServiceEntityConverter<GHPullRequestReviewComment, ReviewComment> {
```

Comment:
Since you called the other entity `PullRequestReview` maybe prefix this also with `PullRequest`?

Reply:
My attempt was to keep the names as concise as possible since they get super long anyways. It does make sense to add it though.